Notify-Tech-Docs/source/documentation/python.md

```diff
@@ -384,7 +382,7 @@ If the request to the client is successful, you will receive the following `dict
   "type": "sms / letter / email", # required string
   "status": "sending / delivered / permanent-failure / temporary-failure / technical-failure", # required string
   "template": {
-    "Version": NUMBER # required string - template version
+    "Version": INTEGER
     "id": `f33517ff-2a88-4f6e-b855-c550268ce08a` # required string - template ID
```

Comment:
More backticks `` ` `` – missed these first time, sorry.

Reply:
Removed these and a few more